### PR TITLE
Bugfix/tracking sheet

### PR DESCRIPTION
**Pull request by kevgliss**
Adds specific runbook, reference, and template UIs in order to make it clearer how to create and associate those types of documents.

Note: I also added a function to all of our ENUMs so we don't have to specify `.value` for all of them. We always use the value so this change simplifies access and prevents us from accidentally forgetting to add `.value`

### CODE REVIEW

File: src/dispatch/document/service.py [hunk 1]
```diff
@@ -22,58 +16,34 @@ def get(*, db_session, document_id: int) -> Optional[Document]:
...
-    return (
-        db_session.query(Document).filter(
-            Document.resource_type == INCIDENT_RESOURCE_INCIDENT_REVIEW_DOCUMENT_TEMPLATE
-        )
-    ).one_or_none()
-
-
-def get_incident_faq_document(*, db_session):
+def get_incident_faq_document(*, db_session, project_id: int):
     """Fetches incident faq docment."""
```
Comment (mvilanova) [inline src/dispatch/document/service.py:32]:
```suggestion
    """Fetches incident faq document."""
```

File: src/dispatch/incident/flows.py [hunk 2]
```diff
@@ -858,23 +858,22 @@ def incident_stable_status_flow(incident: Incident, db_session=None):
 
     # we create a copy of the incident review document template and we move it to the incident storage
     incident_review_document_name = f"{incident.name} - Post Incident Review Document"
-    template = document_service.get_incident_review_template(db_session=db_session)
 
     # incident review document is optional
-    if not template:
+    if not incident.incident_type.review_template_document:
         log.warning("No incident review template specificed.")
```
Comment (mvilanova) [inline src/dispatch/incident/flows.py:864]:
```suggestion
        log.warning("No incident review template specified.")
```

File: src/dispatch/enums.py [hunk 3]
```diff
@@ -5,6 +5,9 @@ class Visibility(str, Enum):
     open = "Open"
     restricted = "Restricted"
 
+    def __str__(self) -> str:
```
Comment (mvilanova) [inline src/dispatch/enums.py:8]: Should we add this function to a parent class that the others would extend from so we don't have to copy and paste it in all of them?

File: src/dispatch/incident/scheduled.py [hunk 4]
```diff
@@ -152,12 +152,12 @@ def daily_report(db_session: SessionLocal, project: Project):
...
                     }
 
-                    if incident.status != IncidentStatus.closed.value:
+                    if incident.status != IncidentStatus.closed:
                         item.update(
                             {
                                 "button_text": "Join Incident",
                                 "button_value": f"{incident.project.organization.slug}-{incident.id}",
-                                "button_action": f"{ConversationButtonActions.invite_user.value}-{incident.status}-{idx}",
+                                "button_action": f"{ConversationButtonActions.invite_user }-{incident.status}-{idx}",
```
Comment (mvilanova) [inline src/dispatch/incident/scheduled.py:160]:
```suggestion
                                "button_action": f"{ConversationButtonActions.invite_user}-{incident.status}-{idx}",
```

File: src/dispatch/static/dispatch/src/api.js [hunk 5]
```diff
@@ -52,8 +52,7 @@ instance.interceptors.response.use(
         store.commit(
           "notification_backend/addBeNotification",
           {
-            text:
-              "Something has gone very wrong, please retry or let your admin know that you received this error.",
+            text: "Something has gone very wrong, please retry or let your admin know that you received this error.",
```
Comment (mvilanova) [inline src/dispatch/static/dispatch/src/api.js:55]: Should we make the message less scary?
```suggestion
            text: "Something has gone wrong, please retry or let your admin know that you received this error.",
```

File: src/dispatch/static/dispatch/src/router/config.js [hunk 6]
```diff
@@ -261,6 +261,24 @@ export const protectedRoute = [
             meta: { title: "Plugins", subMenu: "project", group: "incident" },
             component: () => import(/* webpackChunkName: "plugin-table" */ "@/plugin/Table.vue"),
           },
+          {
+            path: "templates",
+            name: "TemplateTable",
+            meta: { title: "Templates", subMenu: "project", group: "incident" },
+            component: () =>
+              import(
+                /* webpackChunkName: "plugin-table" */ "@/document/template/TemplateTable.vue"
```
Comment (mvilanova) [inline src/dispatch/static/dispatch/src/router/config.js:270]:
```suggestion
                /* webpackChunkName: "template-table" */ "@/document/template/TemplateTable.vue"
```

File: src/dispatch/static/dispatch/src/router/config.js [hunk 7]
```diff
@@ -261,6 +261,24 @@ export const protectedRoute = [
...
+              ),
+          },
+          {
+            path: "references",
+            name: "ReferenceTable",
+            meta: { title: "References", subMenu: "project", group: "incident" },
+            component: () =>
+              import(
+                /* webpackChunkName: "plugin-table" */ "@/document/reference/ReferenceTable.vue"
```
Comment (mvilanova) [inline src/dispatch/static/dispatch/src/router/config.js:279]:
```suggestion
                /* webpackChunkName: "reference-table" */ "@/document/reference/ReferenceTable.vue"
```

File: src/dispatch/static/dispatch/src/router/config.js [hunk 8]
```diff
@@ -294,11 +312,11 @@ export const protectedRoute = [
...
           {
-            path: "documents",
-            name: "DocumentTable",
-            meta: { title: "Documents", subMenu: "project", group: "knowledge" },
+            path: "runbooks",
+            name: "RunbookTable",
+            meta: { title: "Runbooks", subMenu: "project", group: "knowledge" },
             component: () =>
-              import(/* webpackChunkName: "document-table" */ "@/document/Table.vue"),
+              import(/* webpackChunkName: "plugin-table" */ "@/document/runbook/RunbookTable.vue"),
```
Comment (mvilanova) [inline src/dispatch/static/dispatch/src/router/config.js:319]:
```suggestion
              import(/* webpackChunkName: "runbooks-table" */ "@/document/runbook/RunbookTable.vue"),
```

**Review by mvilanova — APPROVED** [review]

File: src/dispatch/enums.py (hunk 3 above)
Comment (kevgliss) [reply src/dispatch/enums.py:8]: Sure, will make the change.

Comment (kevgliss) [reply src/dispatch/enums.py:8]: Will address in separate PR.

**Pull request merged**